src/components/action-sheet/action-sheet.ios.scss

```diff
@@ -103,13 +106,25 @@ $action-sheet-ios-button-cancel-font-weight:            600 !default;
   @include border-radius($action-sheet-ios-border-radius);
   @include margin(null, null, $action-sheet-ios-group-margin-bottom - 2, null);
 
-  overflow: hidden;
+  overflow: scroll;
 
   background: $action-sheet-ios-background;
+
+  -webkit-overflow-scrolling: touch;
+}
+
+.action-sheet-ios .action-sheet-group:first-child {
+  @include margin($action-sheet-ios-group-margin-top, null, null, null);
+
+  flex-shrink: 2;
 }
 
 .action-sheet-ios .action-sheet-group:last-child {
   @include margin(null, null, $action-sheet-ios-group-margin-bottom, null);
+
+  overflow: hidden;
+
+  flex-shrink: 0;
```

Comment:
another way to prevent this node to shrink is to use min-height with the correct value (just one idea, not a big deal).